src/state/cache/thread-mutes.tsx

```diff
@@ -69,7 +69,9 @@ function useMigrateMutes(setThreadMute: SetStateContext) {
         while (!cancelled) {
           const threads = persisted.get('mutedThreads')
 
-          const root = threads.findLast(uri => uri.includes(currentAccount.did))
+          const root = threads
+            .reverse()
+            .find(uri => uri.includes(currentAccount.did))
```

Comment:
I'm not sure why TS got bad about this suddenly, but decided to do this instead of rabbit holing any further on this PR.